addons/analytic/models/analytic_line.py

```diff
@@ -192,3 +193,16 @@ class AccountAnalyticLine(models.Model):
         [('other', 'Other')],
         default='other',
     )
+    fiscal_year = fields.Integer(
+        compute='_compute_fiscal_date',
+        search='_search_fiscal_date',
+    )
+
+    @api.depends('date')
+    def _compute_fiscal_date(self):
+        for line in self:
+            line.fiscal_year = line.date
+
+    def _search_fiscal_date(self, operator, value):
+        fiscalyear_date_range = self.env.company.compute_fiscalyear_dates(fields.Date.today())
```

Comment:
Only inequality are supported. raise NotImplementedError otherwise.

Reply:
By inequality, do you mean ordering operators?